docs/appendices/release-notes/unreleased.rst

```diff
@@ -71,25 +71,24 @@ Deprecations
 Changes
 =======
 
+SQL Statements
+--------------
+
 - Added initial support for cursors. See :ref:`DECLARE <sql-declare>`,
   :ref:`FETCH <sql-fetch>` and :ref:`CLOSE <sql-close>`.
 
-- Added support for ``SET TIME ZONE`` to improve PostgreSQL Compatibility.
-  Timezone will be ignored on the server side.
-
 - Added support for the ``EXISTS`` expression.
```

Comment:
Does the order of this and the next one (correlated sub-query) matter and should be switched?

Reply:
Why should the order matter in your opinion?

Reply:
nevermind, thought that reading top to bottom, sub-queries is a prerequisite for EXISTS.